### PR TITLE
[EOC-382] Unlock editing list after a timeout

### PR DESCRIPTION
**Pull request by alemikolo**

### CODE REVIEW

File: server/sockets/list.js [hunk 1]
```diff
@@ -229,20 +229,48 @@ const updateList = (socket, dashboardViewClients, cohortViewClients) => {
 };
 
 const updateListHeaderState = socket => {
```
Comment (marekrozmus) [inline server/sockets/list.js]: Is this method called only once during initialization? Or on every socket event? Just wondering if `clientLocks` will be created only once.

Comment (alemikolo) [reply server/sockets/list.js]: You're right, it was created multiple times, I had to change it

File: server/sockets/list.js [hunk 2]
```diff
@@ -228,21 +228,47 @@ const updateList = (socket, dashboardViewClients, cohortViewClients) => {
...
+      if (descriptionLock !== undefined) {
+        updatedData.descriptionLock = false;
+      }
+
+      socket.broadcast
+        .to(`sack-${listId}`)
+        .emit(ListHeaderStatusTypes.UNLOCK, updatedData);
+
+      listClientLocks.delete(userId);
+    }, 300000);
```
[the remark this answers is not in the document]
Comment (adamklepacz) [reply server/sockets/list.js]: I did it on EOC-383, you can check it out to place this const in the same place

[the remark this answers is not in the document]
Comment (Ciunkos) [reply server/sockets/list.js]: Yup, just to be sure this gets merged properly :)

**Pull request merged**